src/subhalo_unbind.cpp

```diff
@@ -281,6 +281,23 @@ inline void RefineBindingEnergyOrder(EnergySnapshot_t &ESnap, HBTInt Size, Gravi
 }
 void Subhalo_t::Unbind(const Snapshot_t &epoch)
 { // the reference frame (pos and vel) should already be initialized before unbinding.
+
+  /* We skip already existing orphans */
+  if (!Particles.size())
+  {
+    Nbound = Particles.size();
+    CountParticles(); // Should we remove?
```

Comment:
This seems harmless. If we remove it we'll need to check that we always update nboundtype etc on creating an orphan.

Reply:
For simplicity, I will leave it as is.